packages/dapp-kit-next/examples/react/simple/src/App.tsx

```diff
@@ -3,8 +3,11 @@
 
 import { useStore } from '@nanostores/react';
 import { createDAppKit } from '@mysten/dapp-kit-next';
+import { getFullnodeUrl, SuiClient } from '@mysten/sui/client';
 
-const dAppKit = createDAppKit();
+const dAppKit = createDAppKit({
+	clients: [new SuiClient({ network: 'testnet', url: getFullnodeUrl('testnet') })],
```

Comment:
I wonder if we should make an even more shorthand version of this like `getClient(network)` (probably not a good idea just spit balling)